src/fmu/sumo/explorer/objects/_metrics.py

```diff
@@ -0,0 +1,44 @@
+class Metrics:
+    def __init__(self, search_context):
+        self._search_context = search_context
+        return
+
+    def _aggregate(self, op, **kwargs):
+        aggs = {"agg": {op: {k: v for k, v in kwargs.items() if v is not None}}}
```

Comment:
Without being able to come up with a good alternative myself..
Is there a more descriptive name than `kwargs` we can use?

Reply:
I think it's fairly clear... `kwargs` means `keyword args`, so just whatever keyword arguments are supplied without caring (at this point) what they are.

Common Lisp (of course) does this better, as it allows you to both specify the keyword args and refer to them as a collection.